### PR TITLE
Built

### PR DESCRIPTION
**Pull request by yakup**

### CODE REVIEW

**Comment by paulbjensen** [comment]
Hi,

Thank you for your PR.

I had a look, and I see that node_modules has been commented out of the .gitignore file.

The reason that the node_modules folder is in the .gitignore file is because we don't want to include any of the files and folders inside of the node_modules folder - there could potentially be thousands of files inside that node_modules folder, and having Git track them would be needless - plus it would inflate the git repository's size considerably.

In terms of making sure that the correct versions of the NPM packages are installed, ```npm install``` will find the package-lock.json file, and install dependencies based on the versions specified in that file:

https://docs.npmjs.com/cli/install

In conclusion, it would be best to close the PR, and not merge it in as it would start tracking all of the contents of the node_modules folder.

**Comment by yakup** [comment]
Hi Paul,

I couldn't download dependencies from my work computer due to the company's proxy settings. So I wanted to make a workaround I push node modules from my personal computer. 

I never worked on a forked repository so when I wanted to create a PR it looks like I created a PR for your repository.

Sorry for the inconvenience. 

Best regards,
Yakup
